test/make.sh

```diff
@@ -49,7 +49,10 @@ function docker() {
 function check_terraform() {
   echo "Running terraform validate"
   #shellcheck disable=SC2156
-  find . -name "*.tf" -exec bash -c 'terraform validate --check-variables=false $(dirname "{}")' \;
+  cd in-place-rolling-upgrade || return
```

Comment:
We need to add `$ROOT` here, and why the `||`?

Reply:
The `|| return` or `|| exit` is needed to resolve:
```
In ./test/make.sh line 53:
  cd "${REPO_ROOT}/in-place-rolling-upgrade"
  ^-- SC2164: Use 'cd ... || exit' or 'cd ... || return' in case cd fails.
```